### ISSUE TITLE
Corrupted cobertura file format

### ISSUE THREAD

**Issue by acerbetti** [SlatherOrg/slather#196]
Hi all,
I'm seeing this in the output file produced by slater 2.1 with Xcode7.3.
It was working fine with slather 2.0 and Xcode7.2

Any idea why some rows have hits=21700 and the number is 0?

Thanks for the help 

```
<line number="40" branch="false" hits="110"/>
<line number="41" branch="false" hits="110"/>
<line number="42" branch="false" hits="108"/>
<line number="43" branch="false" hits="108"/>
<line number="0" branch="false" hits="21700.0"/>
<line number="0" branch="false" hits="21700.0"/>
<line number="0" branch="false" hits="21700.0"/>
<line number="0" branch="false" hits="21700.0"/>
<line number="0" branch="false" hits="21700.0"/>
<line number="49" branch="false" hits="0"/>
<line number="50" branch="false" hits="0"/>
<line number="0" branch="false" hits="21700.0"/>
<line number="0" branch="false" hits="21700.0"/>
<line number="0" branch="false" hits="21700.0"/>
<line number="54" branch="false" hits="0"/>
<line number="55" branch="false" hits="0"/>
<line number="56" branch="false" hits="0"/>
<line number="0" branch="false" hits="21700.0"/>
<line number="58" branch="false" hits="108"/>
<line number="59" branch="false" hits="108"/>
<line number="60" branch="false" hits="108"/>
<line number="61" branch="false" hits="110"/>
```

**Comment by ksuther** [comment]
Can you run llvm-cov manually and see what the output for those line look like?

`xcrun llvm-cov show -instr-profile /path/to/Coverage.profdata /path/to/binary`

You can get the paths to the profdata and binary if you run slather with `--verbose`

**Comment by acerbetti** [comment]
This commit https://github.com/SlatherOrg/slather/commit/c95f3d19faa3934e87168826145fd3f6f654d339 fixed my issue. Thanks

**Comment by alanzeino** [comment]
Seeing the same issue:
```
<class name="Foo" filename="path/to/Foo.swift" line-rate="1.0000000000000000" branch-rate="1.0000000000000000" complexity="0.0">
          <methods/>
          <lines>
            <line number="15" branch="false" hits="3"/>
            <line number="16" branch="false" hits="3"/>
            <line number="17" branch="false" hits="3"/>
            <line number="18" branch="false" hits="3"/>
            <line number="20" branch="false" hits="3"/>
            <line number="21" branch="false" hits="3"/>
            <line number="22" branch="false" hits="3"/>
            <line number="23" branch="false" hits="3"/>
            <line number="0" branch="false" hits="70600.0"/>
            <line number="0" branch="false" hits="70600.0"/>
            <line number="0" branch="false" hits="70600.0"/>
            <line number="0" branch="false" hits="70600.0"/>
            <line number="0" branch="false" hits="70600.0"/>
            <line number="0" branch="false" hits="70600.0"/>
            <line number="0" branch="false" hits="70600.0"/>
            <line number="0" branch="false" hits="70600.0"/>
          </lines>
        </class>
```

Any idea when we'll see a new version, @neonichu?

**Comment by neonichu** [comment]
I just shipped 2.2.0 — https://github.com/SlatherOrg/slather/releases/tag/v2.2.0
